kedro/extras/datasets/pandas/sql_dataset.py

```diff
@@ -286,7 +286,19 @@ class SQLQueryDataSet(AbstractDataSet[None, pd.DataFrame]):
         >>>   type: pandas.SQLQueryDataSet
         >>>   sql: "select shuttle, shuttle_id from spaceflights.shuttles;"
         >>>   credentials: db_credentials
-        >>>   layer: raw
+
+    Advance example using the `stream_results` and `chunk_size` option to reduce memorgy usage
```

Comment:
```suggestion
    Advanced example using the `stream_results` and `chunksize` option to reduce memory usage
```

Reply:
Thanks a tonsss for picking this up, I am really bad at this😅.